styles.css

```diff
@@ -15,6 +15,7 @@ h5,
 h6 {
```

Comment:
Unfortunately, I cannot comment on line 1-5, so I will do it here :) it would be better to use only the box-sizing property in the universal-selector (*). Line-height and background-color you can specify in the body.